### ISSUE TITLE
Broken link for Pre-requisites

### ISSUE THREAD

**Issue by robogeekcanada** [ros-industrial/industrial_training#373]
Not sure if raising an issue is the right way to point this out for observations on the training material.

Under the Pre-requisites section, the link for Bruce Eckle's Thinking in C++ is broken 

```
Prerequisites
-------------

C++
~~~~~~~~~~~~~

.. toctree::
   :maxdepth: 1

   MIT Introduction to C++ <http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-096-introduction-to-c-january-iap-2011/assignments/>
   Bruce Eckel Thinking in C++ <http://mindview.net/Books/TICPP/ThinkingInCPP2e.html>
```

Suggested new link: https://web.mit.edu/merolish/ticpp/TicV2.html

**Comment by mariavergo** [comment]
Update to new link: http://chuckallison.com/TICV2/html/TicV2.html

**Comment by hanifz-dr1** [comment]
Fixed in PR #377
